COMETwebapp/COMETwebapp/_Imports.razor

```diff
@@ -0,0 +1,10 @@
+﻿@using System.Net.Http
```

Comment:
Please add File header. Maybe discuss with @samatrhea what it should look like in Blazor/Razor.